data/schedule.json

```
@@ -20,7 +20,7 @@
       "date": "QUA",
       "activities": [
         {
-          "title": "[GI] JIU-JITSU - Todos os Niveís",
+          "title": "[NO-GI] JIU-JITSU - Todos os Niveís",
```

Plz fix this typo everywhere :pray: 

```suggestion
          "title": "[NO-GI] JIU-JITSU - Todos os Níveis",
```